app/models/assignment.rb

```diff
@@ -167,9 +175,10 @@ def run_tests!(params)
     exercise.run_tests! params.merge(extra: extra, test: test, settings: settings)
   end
 
-  def to_resource_h
+  def to_resource_h(options = {})
```

Comment:
I should test this change, too

Reply:
As we aren't using the options for anything else it would be cleaner to use a named param with Organization.current as default, right?